modules/services/images/src/main/res/drawable/ic_play.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="13"
+    android:viewportWidth="10" android:width="18.461538dp" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Is it worth trying to export the image with the size at whole numbers?

I just tried the following:
```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="10dp"
    android:height="13dp"
    android:viewportWidth="10"
    android:viewportHeight="13">
  <path
      android:pathData="M0,1.615C0,1.061 0.38,0.84 0.853,1.124L9.147,6.101C9.618,6.383 9.62,6.84 9.147,7.124L0.853,12.101C0.382,12.383 0,12.156 0,11.61V1.615Z"
      android:fillColor="#000000"/>
</vector>
```

Instead of setting the size to `.size(10.dp, 16.dp)` we can then set it to `.size(10.dp, 13.dp)` so they exactly match.

It's very hard to tell the difference so maybe it doesn't matter.
<img width="1194" alt="Screenshot 2023-07-05 at 11 07 40 am" src="https://github.com/Automattic/pocket-casts-android/assets/308331/d516dda5-8924-4522-b601-a4e153f28549">

Reply:
Good catch. I imported at standard 24dp size and it created fractional viewport sizes. I've updated it in 7d7d1922